css/index.css

```diff
@@ -93,6 +95,57 @@ form button {
   }
 
 
+  /* Update the "Leave a Message" form to use Flexbox or CSS Grid */
+.form-row label {
+    background-color: lightgray;
+}
+
+.form-row input, 
+.form-row textarea,
+.form-row label {
+    width: 100%;
+}
+  
+  .form-wrapper {
+    width: 90%;
+    max-width: 600px;
+    margin: 0 auto;
+    background-color: lightblue;
+
+    display: flex;
+    flex-direction: column;
+
+  }
+
+  .form-row {
```

Comment:
Just to help the readability of the styles, just might wanna make sure there aren't extra spaces to the left of style rules so that it doesn't look like they're nested in something else.
If you're using vs code I believe it has a built in formater you can use, hit ctrl+shift+p, then type "format document" and hit enter with the format document option.

Reply:
Got it. Thank you!